datastore/index.js

```diff
@@ -1,29 +1,45 @@
+const fs = require('fs');
+
+let _data = { 'subscriptions': [] };
+const _datastore_path = 'db.json';
+
 module.exports = {
-  async add({ channelId, githubProjectUrl }) {
-    // TODO save to database
-    console.log(`Subscibe ${channelId} to ${githubProjectUrl}`);
+  async connectToDatastore() {
+    fs.readFile(_datastore_path, (err, data) => {
+      if (err) {
+        if (err.code === 'ENOENT') {
+          // Create local datastore
+          module.exports.commit();
+        }
+        else {
+          throw err;
+        }
+      }
+      else {
+        _data = JSON.parse(data);
+      }
+    });
   },
-  async remove({ channelId, githubProjectUrl }) {
-    // TODO remove from database
-    console.log(`Unsubscribe ${channelId} to ${githubProjectUrl}`);
+  add({ channelId, githubProjectUrl, commit = true }) {
+    _data.subscriptions.push({ channelId: channelId, githubProjectUrl: githubProjectUrl });
+    if (commit) {
+      module.exports.commit();
+    }
```

Comment:
Why not always commit?

Reply:
It was just in case we need to add multiple entries, so, we are not doing writes every time, but all at once.

Reply:
Ok, sounds good~